junction/proposals/management/commands/export_conf_proposals.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+import csv
+import os
+import sys
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+from settings.common import ROOT_DIR
+
+from junction.base.constants import ProposalStatus
+from junction.conferences.models import Conference
+from junction.proposals.models import Proposal, ProposalSectionReviewerVoteValue
+from junction.proposals.permissions import is_conference_moderator
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--conference_slug",
+            default=None,
+            help="Enter the conference slug where to export reviewer votes from",
+        )
+
+        parser.add_argument("--user_id", default=None, help="Enter your user id")
```

Comment:
What is the need of userid ? The command is supposed to be used by the admin from the console so the admin must be logged in to the application server.

Reply:
Not sure, but there are multiple admin users, right?

Reply:
Yes whoever has login access can use this command. No need to check for user id. Usually whoever is managing junction is also the one managing the server. Also user id's are not user friendly